config/locales/en.yml

```diff
@@ -155,27 +153,32 @@ en:
         show:
           forms:
             banner_image:
-              hint: 'To use an image as a masthead background, you should use an image (JPG, GIF or PNG) that is at least 120 pixels tall and 1200 pixels wide. For best results, use an image at least 1800 pixels wide.'
+              hint: "To use an image as a masthead background, you should use an image (JPG, GIF or PNG) that is at least 120 pixels tall and 1200 pixels wide. For best results, use an image at least 1800 pixels wide."
             custom_css:
-              confirm: 'Custom CSS will always override other theming selections. Proceed?'
+<<<<<<< Updated upstream
```

Comment:
not sure what's going on here.

Reply:
All merge conflicts have been resolved and should be good to review.